### DISCUSSION TITLE
Authentication Bearer not matching paths

### DISCUSSION THREAD

**Discussion by ghost** [quarkiverse/quarkus-openapi-generator#303] [Q&A] [answered]
We generate the RestClient interface from a predefined yaml definition with Security Schema set to something similar as:
`securitySchemes:`
 `   user-auth:`
 `     type: http`
 `     description: Security schema using bearer tokens. `
 `     scheme: bearer`
 `     bearerFormat: JWT`
The paths for the restricted endpoit in the yaml is:
`/api/v1/user`
We added the following in the properties file:
`quarkus.rest-client."openapi_user_yaml".url=https://my.server.com/restservice`
`quarkus.openapi-generator."openapi_user_yaml".auth.user-auth.bearer-token=kfdjaslkfdjghlkdsfjhglksdfjhglkjdshfglak`

With this configuration the Authorization header is not added to the outgoing message.  After debugging we found that the CompositeAuthenticationProvider try to watch the path as:
`restservice/api/v1/user to /api/v1/user`

We are using Quarkus version 2.16.3 and quarkus-openapi-generator version 1.3.1

Could you please advise if we miss some configuration in the properties file.  We dont want to at `restservice` path to the yaml definition as this can change and should not be included.

**Comment by hbelmiro** [comment, marked as answer]
Thank you for reporting @arnodtza.
This is really a bug. [I've added a comment to your issue explaining how to reproduce](https://github.com/quarkiverse/quarkus-openapi-generator/issues/304#issuecomment-1516793700).
